### PR TITLE
Support the latest version of Factions.

### PR DESCRIPTION
**Pull request by nikosgram13**
Fix code-style.

### CODE REVIEW

**Comment by jastice** [comment]
Not my preferred style, but it's all yours now! I've added you to the project so you can push directly in future :)

What is your Bukkitdev user so I can add you there?

File: pom.xml [hunk 1]
```diff
@@ -110,19 +110,19 @@
     <dependency>
       <groupId>net.milkbowl.vault</groupId>
       <artifactId>VaultAPI</artifactId>
-      <version>1.6</version>
+      <version>1.5</version>
```
Comment (jastice) [inline pom.xml]: Why downgrade the VaultAPI version?

Comment (nikosgram13) [inline pom.xml]: Because has the same API. Hasn't change anything important. This is why.

**Comment by nikosgram13** [comment]
@jastice my bukkitdev account is nikosgram17 :D

**Pull request merged**